quality/software-checklist.md

```diff
@@ -5,26 +5,34 @@
 - [ ] Does the software have a descriptive name?
 - [ ] Is there a short high-level description of the software?
 - [ ] Is the purpose of the software clear?
+- [ ] Does the software exactly match its requirements?
 - [ ] Is the targeted audience of the software clear?
-- [ ] Does it (and its dependencies) use OSI approved licenses?
+- [ ] Has the software been tested by members of the target audience in respect of its usability?
```

Comment:
Do we want this check in here? Not saying testing it by the target audience is not a good idea, but should that affect the quality of the software?

Reply:
I thought about this some time and in my opinion this is important. Software can be efficient, correct etc., but when it's not usable, it probably won't be accepted - or used.